Cargo.toml

```diff
@@ -41,21 +41,23 @@ fast-math = []
 # experimental nightly portable-simd support
 core-simd = []
 
+bytecheck = ["rkyv/bytecheck"]
+
 [dependencies]
 approx = { version = "0.5", optional = true, default-features = false }
 bytemuck = { version = "1.9", optional = true, default-features = false }
 mint = { version = "0.5.8", optional = true, default-features = false }
 rand = { version = "0.9", optional = true, default-features = false }
 serde = { version = "1.0", optional = true, default-features = false }
-rkyv = { version = "0.7", optional = true, default-features = false }
-bytecheck = { version = "0.7", optional = true, default-features = false }
+rkyv = { version = "0.8", optional = true, default-features = false }
 libm = { version = "0.2", optional = true, default-features = false}
 
 [dev-dependencies]
 # rand_xoshiro is required for tests if rand is enabled
 rand_xoshiro = "0.7"
-# Set a size_xx feature so that this crate compiles properly with --all-targets --all-features
-rkyv = { version = "0.7", default-features = false, features = ["size_32"] }
+# Set a "pointer_width_xx "feature so that this crate compiles properly with --all-targets --all-features.
+# "alloc" is needed to support to_bytes()/deserialize() in tests.
+rkyv = { version = "0.8", default-features = false, features = ["pointer_width_32", "alloc"] }
```

Comment:
This feature was enabled to work around a build error a while ago in https://github.com/bitshifter/glam-rs/commit/abf1a3ca762f5faa4c94998bcb10b0395962fd31, is it still necessary to do this with 0.8?